libcanard/canard.h

```diff
@@ -635,6 +649,54 @@ int8_t canardRxUnsubscribe(CanardInstance* const    ins,
                            const CanardTransferKind transfer_kind,
                            const CanardPortID       port_id);
 
+/// Generate an acceptance filter configuration to accept a specific subject ID.
+///
+/// Complex applications will likely subscribe to more subject IDs than there are
+/// acceptance filters available in the CAN hardware. In this case, the application
+/// should implement filter consolidation. See canardConsolidateAcceptanceFilterConfigs()
+/// as well as the UAVCAN specification for details.
+CanardAcceptanceFilterConfig canardMakeAcceptanceFilterConfigForSubject(const CanardPortID subject_id);
+
+/// Generate an acceptance filter configuration to accept a specific service.
+///
+/// Complex applications will likely subscribe to more service IDs than there are
+/// acceptance filters available in the CAN hardware. In this case, the application
+/// should implement filter consolidation. See canardConsolidateAcceptanceFilterConfigs()
+/// as well as the UAVCAN specification for details.
```

Comment:
Do we really need to restate this fragment multiple times?

Reply:
I'll move it to the top of the acceptance filter API section.